src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -840,11 +848,11 @@ nav.main {
 	text-align: center;
 }
 nav.main .current {
```

Comment:
I think we can remove this ruleset entirely. We don't currently generate a `nav.main .current` AFAICT.

Reply:
We don't seem to have a `nav.main` indeed (but we do have `.current`) so this rule is unused.